shared/config.go

```diff
@@ -42,9 +42,9 @@ func UpdateChainConfig(config *params.ChainConfig) error {
 var (
 	GethRPC                    = "http://localhost:8545"
 	PrivateKey                 = "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
-	BeaconAPI                  = "localhost:3500"
-	BeaconMultiAddress         = "/ip4/0.0.0.0/tcp/13000"
-	BeaconFollowerAPI          = "localhost:3501"
-	BeaconFollowerMultiAddress = "/ip4/0.0.0.0/tcp/13001"
-	ValidatorAPI               = "http://localhost:7500"
+	BeaconAPI                  = "localhost:8000"
+	BeaconMultiAddress         = "/ip4/0.0.0.0/tcp/8000"
+	BeaconFollowerAPI          = "localhost:8001"
+	BeaconFollowerMultiAddress = "/ip4/0.0.0.0/tcp/8001"
```

Comment:
```suggestion
	BeaconMultiAddress         = "/ip4/0.0.0.0/tcp/9000"
	BeaconFollowerAPI          = "localhost:8001"
	BeaconFollowerMultiAddress = "/ip4/0.0.0.0/tcp/9001"
```

Reply:
I think the `MultiAddress` should have the same port configuration we use during the `run_beacon_node.sh` [here.](https://github.com/jimmygchen/eip4844-interop/blob/lighthouse-interop/lighthouse/run_beacon_node.sh#L39)

Reply:
Yeah, this was initially done for testing.
Ideally, we can also get the mutliaddress from the beacon API so we don't have to hardcode anything:
```bash
curl -s http://127.0.0.1:8000/eth/v1/node/identity | jq .data.p2p_addresses
[
  "/ip4/172.25.0.4/tcp/9000/p2p/16Uiu2HAmRWUpDua2Sqh7cb7ftYY3qvudPvxSPF676AjLDxYkMsXQ"
]
```
This way it works for multiple beacon client configurations.